frontend/src/components/Layout/Navbar.tsx

```diff
@@ -16,7 +19,12 @@ const Navbar = () => {
   const darkMode = useDarkModeStore((state) => state.darkMode);
   const { t } = useTranslation(['routes']);
   const auth = useSafeAuth();
-
+  const backendVersion = sessionStorage.getItem('version');
+  useEffect(() => {
+    if (backendVersion != 'frontendVersion') {
```

Comment:
Can this actually happen?

Reply:
The idea was to check if the backend upgraded. Is this the wrong position to check for?

Reply:
Yes, remove it for now, as it is not the scope of this issue.